docs: clarify human added attribution (#1186)
entireio/cli: docs/architecture/attribution.md

3. Calculating agent work (base → shadow minus accumulated user edits)
4. Computing the final percentage

### Attribution Metadata Fields

Attribution is calculated for commits associated with an Entire-tracked agent
session/checkpoint. It does not track keystrokes or exact authorship. The
metrics are inferred from line diffs and hook timing: changes detected before an
agent turn starts, or after the latest checkpoint before commit, are treated as

> user-side changes.

Reviewer: The first-turn case behaves differently: `CheckpointNumber <= 1` is treated as pre-session baseline and subtracted from final human attribution. This wording would distinguish that case from edits made between checkpoints.

```suggestion
Attribution is calculated for commits associated with an Entire-tracked agent
session/checkpoint. It does not track keystrokes or exact authorship. The
metrics are inferred from line diffs and hook timing. Changes made after a
completed checkpoint and detected before the next agent turn, or after the
latest checkpoint before commit, are treated as user-side changes. Changes
already present before the first agent turn are treated as a pre-session
baseline and excluded from the final human contribution counts.
```

- `agent_lines` - agent-attributed added lines that remain in the commit
- `agent_removed` - agent-attributed deletions that remain deleted in the commit
- `human_added` - user-side additions that are not paired with removals as modifications

> - `human_modified` - estimated replacement-style user-side changes, calculated from paired additions and removals

Reviewer: This is an aggregate count heuristic: the implementation uses `min(totalUserAdded, totalUserRemoved)`, so additions and removals need not be in the same hunk or file.

```suggestion
- `human_modified` - aggregate estimate calculated as the smaller of eligible user-side additions and removals; pairing is not tracked per hunk or file
```

> - `human_removed` - user-side removals that are not paired with additions as modifications

Reviewer: The current field is narrower than all user-side removals: post-checkpoint deletions in non-agent files affect `total_lines_changed` but are not added to `HumanRemoved`. This suggestion describes the implemented scope; alternatively, the calculation could be fixed and covered by a regression test.

```suggestion
- `human_removed` - estimated unpaired user-side removals captured between prompts or after the latest checkpoint in agent-touched files
```

> - `total_committed` - legacy net-additions metric

Reviewer: This is not always literal net additions: when the calculated value is non-positive, the implementation falls back to the agent-added count.

```suggestion
- `total_committed` - legacy additions-focused compatibility metric; not guaranteed to equal the commit's literal net line count
```
